### PR TITLE
Fixed: LA Report - Missing Sickness for Pool/Bank/Agency

### PR DESCRIPTION
**Pull request by rchitranshi**
https://trello.com/c/ftBo3hQ4

### CODE REVIEW

**Comment by rchitranshi** [comment]
[LA Report - Missing Sickness for Pool/Bank/Agency](https://trello.com/c/ftBo3hQ4/55-la-report-missing-sickness-for-pool-bank-agency)

File: server/routes/reports/localAuthorityReport/user.js [hunk 1]
```diff
@@ -1079,7 +1087,7 @@ const updateStaffRecordsSheet = (
     sharedStrings,
     sst,
     sharedStringsUniqueCount
-) => {
+) => {;
```
Comment (aaron-russell) [inline server/routes/reports/localAuthorityReport/user.js]: There's a random semi colon here

Comment (pjshumphreys) [inline server/routes/reports/localAuthorityReport/user.js]: Please remove this extra semicolon

Comment (rchitranshi) [reply server/routes/reports/localAuthorityReport/user.js]: Removed.. Please check now.

**Comment by pjshumphreys** [comment]
Nice work dude 👍

**Comment by aaron-russell** [comment]
Looks good now

**Pull request merged**